openff/evaluator/protocols/forcefield.py

```diff
@@ -1052,3 +1063,54 @@ def _execute(self, directory, available_resources):
             )
 
         super(BuildTLeapSystem, self)._execute(directory, available_resources)
+
+
+@workflow_protocol()
+class BuildFoyerSystem(TemplateBuildSystem):
+    """Parameterize a set of molecules with a Foyer force field source"""
+
+    def _parameterize_molecule(self, molecule, force_field_source, cutoff):
+        """Parameterize the specified molecule.
+
+        Parameters
+        ----------
+        molecule: openff.toolkit.topology.Molecule
+            The molecule to parameterize.
+        force_field_source: FoyerForceFieldSource
+            The foyer source which describes which parameters to apply.
+
+        Returns
+        -------
+        openmm.System
+            The parameterized system.
+        """
+        import mdtraj as md
+        from foyer import Forcefield
+        from openff.interchange import Interchange
+        from openff.toolkit import Topology
+
+        topology: Topology = molecule.to_topology()
+        topology.mdtop = md.Topology.from_openmm(topology.to_openmm())
```

Comment:
Is this still needed? I thought I got rid of `Topology.mdtop` in recent versions (0.3.0+) of Interchange

Reply:
Seems like it is not needed, I was going by the Interchange documentation https://docs.openforcefield.org/projects/interchange/en/stable/using/migrating.html

Reply:
Those docs are probably outdated - thanks for bringing this to my attention!